runtime/lua/vim/highlight.lua

```diff
@@ -75,21 +85,29 @@ end
 
 local yank_ns = api.nvim_create_namespace('hlyank')
 local yank_timer
---- Highlight the yanked region
+--- Temporally highlight the last yanked region. This function can be used in
+--- conjunction with |TextYankPost| autocmd to automatically highlight just
+--- yanked region:
+--- <pre>lua
+---   vim.api.nvim_create_autocmd("TextYankPost", {
```

Comment:
```suggestion
vim.api.nvim_create_autocmd("TextYankPost", {
```

Reply:
Do you mean this?
```suggestion
--- vim.api.nvim_create_autocmd("TextYankPost", {
```

Reply:
Sorry, the Github diffview tricked me into seeing an empty line where there was none. Disregard, please.